source/usage-examples/struct-tagging.txt

```diff
@@ -0,0 +1,67 @@
+==========================
+Set Custom Driver Behavior
```

Comment:
I'm not satisfied with this title, so very open to suggestions. I decided to go with Chris's suggestion and just center this Usage Example around using struct tags for custom BSON field names, as an ``omitempty`` example will be in the fundamentals pg.

Reply:
maybe `Rename Struct Fields`? Depends on what the primary use case is here.